addons/account/views/report_invoice.xml

```diff
@@ -256,13 +256,11 @@
                                         </div>
                                     </t>
                                 </div>
-                                <div class="mb-3" t-if="o.move_type in ('out_invoice', 'in_refund') and o.payment_reference">
-                                    <p name="payment_communication">
-                                        Payment Communication: <span class="fw-bold" t-field="o.payment_reference">INV/2023/00001</span>
-                                        <t t-if="o.partner_bank_id">
-                                            <br/> on this account: <span t-field="o.partner_bank_id" class="fw-bold"/>
-                                        </t>
-                                    </p>
+                                <div class="mb-3" t-if="o.move_type in ('out_invoice', 'in_refund') and o.payment_reference"  name="payment_communication">
+                                    Payment Communication: <span class="fw-bold" t-field="o.payment_reference">INV/2023/00001</span>
+                                    <t t-if="o.partner_bank_id">
+                                        <br/> on this account: <span t-field="o.partner_bank_id" class="fw-bold"/>
```

Comment:
what is the reason behind this change in stable version? `<p/>` is named (`payment_communication`) so probably used in existing xpaths

Reply:
My bad, I was misunderstanding how template works :)